doc/errorcodes.rst

```diff
@@ -0,0 +1,80 @@
+|| ``C003``  |  Wrongly split compounds united. Should be corrected.                                        |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C004``  |  Duplicated word marked as a possible error. Should be pointed out but not deleted.          |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C005``  |  Possible split compound, depends on meaning/PoS chosen by parser.                           |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C006``  |  A part of a compound word is wrong.                                                         |
++------------+----------------------------------------------------------------------------------------------+
+|| ``C007``  |  A multiword compound such as "skóla-og frístundasvið" correctly split up                    |
++------------+----------------------------------------------------------------------------------------------+
+|| ``E001``  |  Unable to parse sentence, possibly something wrong with it.                                 |
++------------+----------------------------------------------------------------------------------------------+
+|| ``E004``  |  The sentence is probably not in Icelandic.                                                  |
++------------+----------------------------------------------------------------------------------------------+
+|| ``N001``  |  Wrong quotation marks                                                                       |
++------------+----------------------------------------------------------------------------------------------+
+|| ``N002``  |  Three periods should be an ellipsis                                                         |
++------------+----------------------------------------------------------------------------------------------+
+|| ``N003``  |  Informal combination of punctuation marks (??!!)                                            |
++------------+----------------------------------------------------------------------------------------------+
+|| ``P_xxx`` |  Phrase error codes. Description for each type is provided within error annotations.         |
++------------+----------------------------------------------------------------------------------------------+
+|| ``S001``  |  Common errors picked up by unique_errors. Should be corrected.                              |
++------------+----------------------------------------------------------------------------------------------+
+|| ``S002``  |  Errors handled by spelling.py. Corrections should possibly only be suggested.               |
++------------+----------------------------------------------------------------------------------------------+
+|| ``S003``  |  Erroneously formed word forms picked up by ErrorForms. Should be corrected.                 |
```

Comment:
"Erroneously formed words" is better. "picked up by ErrorForms" is an implementation detail.

Reply:
Updated description.